libs/efivar/patches/002-musl-compat.patch

```diff
@@ -0,0 +1,187 @@
+--- a/src/compiler.h
```

Comment:
I found this patch at https://git.alpinelinux.org/aports/tree/main/efivar/musl-compat.patch.
Please keep his patch header here.

Reply:
Sure. Header was lost during patch refreshing for quilt compatibility (dirty patches...)